README.md

```diff
@@ -8645,6 +8647,29 @@ There is an exception, though: *experimental features*, marked with the "**:fire
 Those experimental features are not protected by semantic versioning, and may break between two minor releases of the library. To help them becoming stable, [your feedback](https://github.com/groue/GRDB.swift/issues) is greatly appreciated.
 
 
+### SQLite error 21 "wrong number of statement arguments" with LIKE queries
+
+You may get the error "wrong number of statement arguments" when executing a LIKE query similar to:
+
+```swift
+let name = textField.text
+try dbQueue.read { db in
+    try db.execute(sql: "SELECT * FROM students where title like '%?%'", arguments: [name])
```

Comment:
One does not use `execute` with SELECT queries. I suggest instead:

```swift
let name = textField.text
let players = try dbQueue.read { db in
    try Player.fetchAll(db, sql: "SELECT * FROM player WHERE name LIKE '%?%'", arguments: [name])
}
```

Stylewise, note:

- Usage of the Player type, one of the few example types used in GRDB documentation.
- Uppercase SQL keywords, as everywhere in GRDB documentation.
- Singular table name, as [recommended](https://github.com/groue/GRDB.swift/blob/master/README.md#create-tables).